payload-types.ts

```diff
@@ -15,6 +15,7 @@ export interface Config {
     media: Media;
     person: Person;
     'sports-team': SportsTeam;
+    studentGuide: StudentGuide;
```

Comment:
use slug here, to be consistent

Reply:
this file is generated by Payload.

Reply:
Right, but we should be consistent in the text format of the slugs, if we follow the 'sports-team', then 'student-guide', otherwise, change everything to camelCase